service/rosetta.go

```diff
@@ -14,6 +14,6 @@ var NodeVersion = fmt.Sprintf(
 )
 
 const (
-	MiddlewareVersion = "0.1.21"
+	MiddlewareVersion = "0.1.22"
```

Comment:
Can you remind me of this variable?

Reply:
It's the version for Rosetta App

Reply:
We update it sequentially for every rosetta release

Reply:
Ahhh awesome, thanks